jobs/kola-kubernetes.Jenkinsfile

```diff
@@ -0,0 +1,80 @@
+def streams
+node {
+    checkout scm
+    streams = load("streams.groovy")
+}
+
+properties([
+    pipelineTriggers([]),
+    parameters([
+      choice(name: 'STREAM',
+             // list devel first so that it's the default choice
+             choices: (streams.development + streams.production + streams.mechanical),
+             description: 'Fedora CoreOS stream to test'),
+      string(name: 'VERSION',
+             description: 'Fedora CoreOS Build ID to test',
+             defaultValue: '',
+             trim: true),
+      string(name: 'ARCH',
+             description: 'Target architecture',
+             defaultValue: 'x86_64',
+             trim: true),
+      string(name: 'S3_STREAM_DIR',
+             description: 'Override the Fedora CoreOS S3 stream directory',
+             defaultValue: '',
+             trim: true),
+      string(name: 'COREOS_ASSEMBLER_IMAGE',
+             description: 'Override the coreos-assembler image to use',
+             defaultValue: "coreos-assembler:main",
+             trim: true),
+      string(name: 'FCOS_CONFIG_COMMIT',
+             description: 'The exact config repo git commit to run tests against',
+             defaultValue: '',
+             trim: true),
+    ]),
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+])
+
+currentBuild.description = "[${params.STREAM}][${params.ARCH}] - ${params.VERSION}"
+
+def s3_stream_dir = params.S3_STREAM_DIR
+if (s3_stream_dir == "") {
+    s3_stream_dir = "fcos-builds/prod/streams/${params.STREAM}"
+}
+
+try { timeout(time: 60, unit: 'MINUTES') {
+    cosaPod(image: params.COREOS_ASSEMBLER_IMAGE,
+            memory: "256Mi", kvm: false,
+            secrets: ["aws-fcos-builds-bot-config", "aws-fcos-kola-bot-config"]) {
+
+        stage('Fetch Metadata') {
+            def commitopt = ''
+            if (params.FCOS_CONFIG_COMMIT != '') {
+                commitopt = "--commit=${params.FCOS_CONFIG_COMMIT}"
+            }
+            shwrap("""
+            export AWS_CONFIG_FILE=\${AWS_FCOS_BUILDS_BOT_CONFIG}/config
+            cosa init --branch ${params.STREAM} ${commitopt} https://github.com/coreos/fedora-coreos-config
+            cosa buildfetch --artifact=ostree --artifact=qemu --build=${params.VERSION} \
+                --arch=${params.ARCH} --url=s3://${s3_stream_dir}/builds
+            """)
+        }
+
+        fcosKola(cosaDir: env.WORKSPACE,
+                 build: params.VERSION, arch: params.ARCH,
+                 extraArgs: "--tag k8s",
+                 skipBasicScenarios: true,
+                 platformArgs: """-p=aws \
```

Comment:
might be worth adding a comment here about why we chose AWS. I assume it's because it's the only platform we have images pre-uploaded to (AWS and GCP right now) that also has `x86_64` **AND** `aarch64`.

Reply:
Added comment.